CrewChiefV4/Events/Fuel.cs

```diff
@@ -1218,7 +1261,7 @@ public override int resolveMacroKeyPressCount(String macroName)
 
             // limit the number of key presses to 200 here, or fuelCapacity
             int fuelCapacityInt = (int)fuelCapacity;
-            if (fuelCapacityInt > 0 && fuelCapacityInt < litresToEnd)
+            if (fuelCapacityInt > 0 && fuelCapacityInt-currentFuel < litresToEnd)
```

Comment:
feel free to ignore my anal comments, but spaces before after operator, please

Reply:
is this correct? The logic here is "if, after adding the calculated required amount of fuel, we will still need another stop, then warn about that"

So the idea was to assume that the max amount of possible fuel after the stop is a full tank, and if a full tank isn't enough to get the end we play this message.

I'm not sure i understand why you're subtracting the current fuel from this check